src/assets/images/ix_user-profile-filled.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
please follow the same naming convention. In this case the whole file name should be in kebab case and you shouldn't mix cases